meta-facebook/yv35-cl/src/platform/plat_hook.c

```diff
@@ -405,3 +406,39 @@ bool post_ltc4282_read(uint8_t sensor_num, void *args, int *reading)
 	sval->fraction = (val - sval->integer) * 1000;
 	return true;
 }
+
+bool pre_intel_dimm_i3c_read(uint8_t sensor_num, void *args)
+{
+	ARG_UNUSED(args);
+
+	if (get_post_status() == false) {
+		// BIC can't check DIMM present status, return true to keep do sensor initial
+		return true;
+	}
+
+	dimm_pre_proc_arg *pre_proc_args = (dimm_pre_proc_arg *)args;
+	if (pre_proc_args->is_present_checked == true) {
+		return true;
+	}
+
+	uint8_t dimm_id = -1;
+	bool ret = false;
+
+	dimm_id = sensor_num_map_dimm_id(sensor_num);
+	if (dimm_id < 0) {
```

Comment:
Check here isn't needed since dimm_id is a uint8_t.

Reply:
Hi @garnermic 
Thanks for your comments.
Based on your suggestions to modify the code and force update this branch.
We add a new definition "DIMM_ID_UNKNOWN" to check if dimm_id can't map.
Please help review it again, thank you.